containers/wordpress/wp_base/https/files/nginx.conf.in

```diff
@@ -0,0 +1,122 @@
+worker_processes  96; # Depends on #vcpus
```

Comment:
It would be better to make this a variable. Also if we use NGNIX Async, do we need the nginx.conf.in file?

Reply:
Right now the entrypoint script will replace this at run time to match # of vcpus on the system.  I think the conf.in file is still used per the internal implementation

Reply:
To add to my response, the crypto reference stack currently has scripts to manage nginx execution, this script has config values passed to it as parameters and then behind the scenes nginx still uses a config file.